# Thesisscripts
Scripts To BE USED

Structure Information

Files 

align.ipynb - gives the alignment score 
input file - gpx4.faa 
blast.ipynb - gives an xml file with results
blosum.ipynb - could only create an alignment of a range of amino acids and score of it between two sequences, but not the matrix, have to check 
glutathione.faa - has all sequences of glutathione in all species from uniprot
extract.ipynb - gived info from glutathione.faa index, record.id, len(record.seq), len(record.features and a fasta also, reads the fasta from the file, but no specification of organism
